timeseries/src/autogluon/timeseries/models/autogluon_tabular/direct_tabular.py

```diff
@@ -386,7 +388,7 @@ def predict(
         # Predict for batches (instead of using full dataset) to avoid high memory usage
         batches = features.groupby(np.arange(len(features)) // self.PREDICTION_BATCH_SIZE, sort=False)
         predictions = pd.concat([self.tabular_predictor.predict(batch) for _, batch in batches])
-        predictions.set_index(data_future.index, inplace=True)
+        predictions.index = data_future.index
```

Comment:
Wow, that is very subtle. I'm surprised Series doesn't have `.set_index`, seems like a strange difference in API. Good catch!